extensions/historical-imagery/schema.json

```diff
@@ -17,7 +17,7 @@
             "properties": {
               "type": "object",
               "$comment": "Require fields here for Item Properties.",
-              "required": ["platform", "instruments", "mission"]
+              "required": ["platform", "mission"]
```

Comment:
We should probably have an item test to make sure removing the `properties/instruments` doesn't cause an error.

Reply:
I've added a test that passes with removing properties/instruments doesn't cause an error.
I don't quite follow about failing "for the right reason without this change"?

Reply:
> I don't quite follow about failing "for the right reason without this change"?

It's one of the steps of TDD:

1. Write a failing test (before changing *anything* else in the code base)
2. Run the test
3. Check that the test failed _for the reason you expected_ (in this case, that it results in a "must have required property 'aerial-instruments'" error message). If it succeeds then the test isn't verifying what you expected. If it fails for some _unexpected_ reason then it requires more research to determine whether it's even possible to make this change without doing something else first, like those "not valid in any of the specified sub-schemas" errors we were getting when using `oneOf`).
3. Change the production code until the test passes
4. Simplify
5. Commit
6. Repeat